src/core/include/openvino/op/adaptive_avg_pool.hpp

```diff
@@ -15,7 +15,6 @@ namespace v8 {
 class OPENVINO_API AdaptiveAvgPool : public Op {
 public:
     OPENVINO_OP("AdaptiveAvgPool", "opset8");
-    BWDCMP_RTTI_DECLARATION;
```

Comment:
Before the merge we need to revert these changes.

Reply:
Reverted